src/Summary.jsx

```diff
@@ -132,6 +134,11 @@ class Summary extends React.Component {
               </Link>
               <h1>Sobre el área</h1>
               <div className="line" />
+              <h5 className="hectareas vulne">
+                Vulnerabilidad:
+                {' '}
+                <b>{numberWithCommas(Number(activeArea.vulnerability).toFixed(2))}</b>
```

Comment:
Right now if the property doesn't exist it shows "NaN". I think it's better to use the ternary operator and show a default message in case activeArea.vulnerability is null or undefined

Reply:
Yes, I changed to validate it. Thanks